### ISSUE TITLE
Feature suggestion: Temporary whitelist

### ISSUE THREAD

**Issue by binaryanomaly** [pi-hole/pi-hole#4438]
Hi,

I think it would be nice to have the possibility to temporarily whitelist a domain because you need it only once.
Little Snitch on Mac has a feature like this and I find it extremely valuable.

Thought I'd suggest this.

Thanks for considering.
-b

**Comment by yubiuser** [comment]
Please submit all feature request to: https://discourse.pi-hole.net/

In this case, see:
https://discourse.pi-hole.net/t/add-a-temporarily-allow-feature/3947
